nosepipe.py

```diff
@@ -147,12 +147,16 @@ def __call__(self, result):
             useshell = True
 
         self.logger.debug("Executing %s", " ".join(argv))
-        popen = subprocess.Popen(argv,
-                                 cwd=self._cwd,
-                                 stdout=subprocess.PIPE,
-                                 stderr=subprocess.STDOUT,
-                                 shell=useshell,
-                                 )
+        try:
+            popen = subprocess.Popen(argv,
+                                     cwd=self._cwd,
+                                     stdout=subprocess.PIPE,
+                                     stderr=subprocess.STDOUT,
+                                     shell=useshell,
+                                     )
+        except OSError, e:
```

Comment:
I'm running with `nosetests`.  I've setup a travis job to show you the failure I'm getting here: 

https://travis-ci.org/mdprewitt/nosepipe/jobs/51232640

I'm also messing around with seeing how I can fix this here:

https://github.com/mdprewitt/nosepipe/pull/1